### PR TITLE
lets-talk commit

### PR DESCRIPTION
**Pull request by Dorenus**
Lets-talk section

### CODE REVIEW

**Review by liepsneletech — CHANGES_REQUESTED** [review]
Everything looks nice, just need to change the text color of these elements to #454545
https://prnt.sc/BAeoiNDa_JYc

File: css/pages/home.css [hunk 1]
```diff
@@ -206,6 +206,26 @@
...
+}
+.title-description {
+    font-size: 18px;
+    font-weight: 400;
+    margin: 40px 0;
+}
+
+.feature-box {
+    display: flex;
+}
```
Comment (zilva149) [inline css/pages/home.css:228]: This is unnecessary.

**Review by zilva149 — COMMENTED** [review]
Looks good, just minor change.

File: css/pages/home.css (hunk 1 above)
Comment (Dorenus) [reply css/pages/home.css]: Thanks, deleted those lines of code.

**Review by liepsneletech — APPROVED** [review]

**Review by zilva149 — APPROVED** [review]

**Review by EvgenySivaykin — APPROVED** [review]

**Pull request merged**